lib/generators/gql_show/gql_show_generator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class GqlShowGenerator < Rails::Generators::NamedBase
+  source_root File.expand_path('templates', __dir__)
+
+  def create_gql_file
+    template 'exemple_spec.rb', File.join('spec/graphql/queries/types', "query_type_#{file_name.underscore}_spec.rb")
+    template 'exemple.graphql.rb', File.join('spec/fixtures/graphql', "#{file_name.underscore}.graphql")
```

Comment:
en anglais ça s'écrit plutôt `example`
```suggestion
    template 'example_spec.rb', File.join('spec/graphql/queries/types', "query_type_#{file_name.underscore}_spec.rb")
    template 'example.graphql.rb', File.join('spec/fixtures/graphql', "#{file_name.underscore}.graphql")
```
(penser à renommer les fichiers également)